frontend/src/pages/team/routes.js

```diff
@@ -174,7 +180,7 @@ export default [
                     {
                         name: 'team-overview',
                         path: 'overview',
-                        redirect: { name: 'Applications' }
+                        redirect: { name: 'team-home' }
```

Comment:
Remove this option now, we have the new home page, and should update the "home page" URL to 'overview'. Then anyone that had it bookmarked (which is why this was left in) now gets the new Home experience instead.